api/v1/user/api/views.py

```
@@ -0,0 +1,30 @@
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.permissions import IsAuthenticated
+
+from .serializers import RegistrationSerializer
+
+
+@api_view(['POST'])
+def registration_view(request):
+	if request.method == 'POST':
+		serializer = RegistrationSerializer(data=request.data)
+		data = {}
+		if serializer.is_valid():
+			user = serializer.save()
+			data['response'] = "Successfully registered new user"
+			data['email'] = user.email
+			data['username'] = user.username
+			data['program'] = user.program
+			data['term'] = user.term
+		else:
+			data = serializer.errors
```

Should return an error message with code